--- a/mqx/mqx/source/psp/coldfire/psp_prv.inc
+++ b/mqx/mqx/source/psp/coldfire/psp_prv.inc
@@ -274,7 +274,7 @@
         lea.l   -4*16(sp),sp            ; make room on the stack
         movem.l d3-d7,4*3(sp)           ; save regs (can optimize and
         movem.l a2-a6,4*10(sp)          ; not save d0,d1,d2,a0,a1)
-        move.w  #$4000,d0               ; set coldfire stack frame
+        move.w  #0x4000,d0              ; set coldfire stack frame
         swap    d0
         move.w  sr,d0
         move.l  d0,4*15(sp)             ; save SR and format
